README.md

```diff
@@ -16,6 +16,13 @@ The Zotero translation server lets you use [Zotero translators](https://www.zote
 
 `npm start`
 
+## Running via docker
+
+```
+docker build -t translation-server:latest .
+docker run -it -d -p 1969:1969 translation-server:latest
```

Comment:
Can we just remove the `:latest` tag altogether? I don't think it's necessary.

Also, for the `run` line, we don't need `-it`, and we should add `--rm` so stopped containers are cleared automatically.

Reply:
Ok, i just fixed it.